README.md

```diff
@@ -1,5 +1,10 @@
-# Open Source Programs (Backend)
 ![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
```

Comment:
```suggestion
# Open Source Programs (Backend)

![Build Status](https://github.com/anitab-org/open-source-programs-backend/workflows/Tests%20Build/badge.svg)
```

Reply:
Nice suggestion. I wasn't sure of the preference, thank you.